app/controllers/comments_controller.rb

```diff
@@ -5,6 +5,10 @@ class CommentsController < ApplicationController
   def index
     @comments = @commentable.comments
   end
+	
```

Comment:
have a look at the basic controller spec scaffolding:
https://github.com/rspec/rspec-rails/blob/master/lib/generators/rspec/scaffold/templates/controller_spec.rb